Source/IPC/SimReadFromFile.c

```diff
@@ -29,209 +29,209 @@ void ReadFromFile(FILE *StateFile, long EchoEnabled)
             RequestTimeRefresh = 1;
          }
 
-            if (sscanf(line,"Orb[%ld].PosN = %le %le %le",
-               &Iorb,
-               &DbleVal[0],
-               &DbleVal[1],
-               &DbleVal[2]) == 4) {
-               Orb[Iorb].PosN[0] = DbleVal[0];
-               Orb[Iorb].PosN[1] = DbleVal[1];
-               Orb[Iorb].PosN[2] = DbleVal[2];
-            }
+         if (sscanf(line,"Orb[%ld].PosN = %le %le %le",
```

Comment:
Wondering if it would be more efficient to turn all these 'if' statements into an 'if, else-if'?  Only one should match per line processed (had come across this separately looking in the socket processing code).

BTW, these same changes may be applicable to the SimReadFromSocket.c (and others?)...know you were just getting rid of compiler warnings, but if these are "common", may want the line scanning in a separate function instead of duplicated in multiple files...

Reply:
Those are some good suggestions.